cmd/edgectl/aes_install.go

```diff
@@ -809,6 +901,19 @@ Timed out waiting for the load balancer's IP address for the AES Service.
 
 const tryAgain = "If this appears to be a transient failure, please try running the installer again. It is safe to run the installer repeatedly on a cluster."
 
+const abortExisting = `
+This tool does not support upgrades/downgrades at this time.
+
+Aborting the installer to avoid corrupting an existing installation of AES.
+`
+
+// TODO: Explain how to remove existing CRDs? Or do something smarter?
```

Comment:
Yes, I think we should have instructions for removing the existing CRDs and resuming using the installer 👍

Reply:
I couldn't come up with those instructions quickly, so I punted to a TODO. Thoughts? 

- `kubectl delete ambassador-crds` doesn't work
- `kubectl delete crd -l product=aes` only works if the CRDs are installed using our YAML. 
- `kubectl delete $(kubectl get crd -o name | grep "[.]getambassador[.]io$")` doesn't work on Windows

Reply:
I would do `kubectl delete crd -l product=aes`. Assuming the listed CRDs are also visible in the terminal output, _most_ users should be able to figure it out.

Reply:
Oh, the other issue is that deleting the CRDs also deletes and CRs of those types. Need to warn about that!